packages/edit-post/src/index.native.js

```diff
@@ -31,5 +31,5 @@ export function initializeEditor( id, postType, postId ) {
 
 	editorInitialized = true;
 
-	render( <Editor postId={ postId } postType={ postType } />, id );
+	render( <EditorPreview postId={ postId } postType={ postType } />, id );
```

Comment:
At first, I found a bit confusing that we are rendering an `EditorPreview` component instead of `Editor` and then within `EditorPreview` we are deciding which to render. I wonder if it'd make sense to have that logic that we have now in `editor-preview.native.js` to be here instead since its just a couple of lines of code. What do you think?

Reply:
We could have in this same file a functional component e.g:

```
function EditorMode( props ) {
	const { initialHtml, editorMode, postId, postType } = props;

	if ( editorMode === 'preview' ) {
		return <Preview blocks={ rawHandler( { HTML: initialHtml } ) } />;
	}

	return <Editor postId={ postId } postType={ postType } { ...props } />;
}
```

And then:

```suggestion
	render( <EditorMode postId={ postId } postType={ postType } />, id );
```

Reply:
Thanks for the tips on this one @geriux!